LDM-732.tex

```diff
@@ -26,19 +26,14 @@
 
 \setDocCompact{true}
 
-\title[Network Verification Document]{Vera C. Rubin Observatory Network Verification Document}
+\title[Network Verification Elements]{Network Verification Elements}
 
 \author{Joshua Hoblitt, Jeff Kantor}
 \setDocRef{\lsstDocType-\lsstDocNum}
 \setDocDate{\today}
 
 \setDocAbstract {
-The Vera C Rubin Observatory Network Verification Document (VNVD) and associated JIRA V\&V
-Project define the flow-down of specifications from higher level documents to the LSST
-Observatory Network (as defined in LSE-78 LSST Observatory Network), and the methods and
-resources that will be used to verify that the networks have met the specifications satisfactory
-for accepting the Summit Network into DM Subsystem Integration Test (DMSSIT) and LSST
-System Integration Test (SIT)
+The Network Verification Elements document and associated \href{https://jira.lsstcorp.org/projects/LVV/}{LSST Verification and Validation (LVV)} Jira project define the flow-down of specifications from higher level documents to the Observatory Network (as defined in \citeds{LSE-61} Data Management System (DMS) Requirements), as well as identifying the methods and resources that will be used to verify that network requirements have been met. \jira{LVV}
```

Comment:
define the flowdown ->  present the flow-down 
as identifying the methods -> as defining the methods 

the flowdown is **defined** in the requirements documents, this document **presents** those requirements  and defines the additional the methods and resources that will be used to verify that network requirements have been met.

Reply:
Done.